README.md

```diff
@@ -0,0 +1,51 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Creative Commons - Conference Summit 2015
+
+This project is a skill demonstration of a real work demand requested by Microverse's Capstone Project.
+
+Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07) 
+
+
+
+## Demo
+Get the live link [here](https://fabianofrank.github.io/conference-page/)
+
+## Built With
+
+- JS
+- HTML and CSS
+- GIT
+
+## Getting Started
+To get a local copy up and running follow these [steps](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+
```

Comment:
- [ ] It would be better if you add steps about how to start the project locally. Kindly add this. 

😄 👍